### PR TITLE
[66] Style the Contact Us Page using CSS

### PR DESCRIPTION
**Pull request by queensaldam**
## Task: `Contact us page styling using CSS`
- `Style the contact us page`

### CODE REVIEW

**Comment by queensaldam** [comment]
contact us desktop page
contact us tablet
contact us mobile

**Comment by queensaldam** [comment]
contact us desktop page
contact us tablet
contact us mobile

File: submissions/damilolaSalami/html-form/contactUs.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/damilolaSalami/html-form/contactUs.html:1]: kindly work on your existing contact form page. Make changes there. do not create a new file

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/damilolaSalami/html-form/contactUs.html (hunk 1 above)
Comment (adaugochi) [reply submissions/damilolaSalami/html-form/contactUs.html:1]: kindly delete this file

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — CHANGES_REQUESTED** [review]
All images should be placed inside the `images` folder that is in your `html` folder

**Comment by queensaldam** [comment]
> All images should be placed inside the `images` folder that is in your `html` folder

Please I don't understand what I am to do

**Review by adaugochi — APPROVED** [review]

**Pull request merged**